src/adapters/codemirror-6-adapter.ts

```diff
@@ -84,13 +84,14 @@ export class CodeMirror6Adapter implements AdapterInterface {
     this.lastContentChecked = this.currentContentChecking;
   }
 
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   registerCheckCall(_checkInfo: Check) {
     return;
   }
 
   private lookupMatchesOrThrow<T extends Match>(matches: T[]): AlignedMatch<T>[] {
     const alignedMatches = lookupMatches(this.lastContentChecked!, this.getContent(), matches, 'TEXT');
-    if (_.isEmpty(alignedMatches)) {
+    if (alignedMatches.length === 0) {
```

Comment:
The lodash `_.isEmpty(null)` would return true ([docs](https://lodash.com/docs#isEmpty)).  So I would suggest to replace it with `if(!alignedMatches || alignedMatches.length === 0)`